__tests__/tripRequestTest/tripRequestReducer.test.js

```diff
@@ -0,0 +1,32 @@
+import tripRequestReducer from '../../src/reducers/tripRequestReducer';
+describe('signIn reducer test ', () => {
+	it('should return success message', () => {
+		const action = { type: 'REQUEST_TRIP_SUCCESS' };
+		const initialState = {
+			error: '',
+			message: undefined,
+			tripCreated: undefined,
+		};
+		expect(tripRequestReducer({}, action)).toEqual(initialState);
+	});
+	it('should return error', () => {
+		const action = { type: 'REQUEST_TRIP_FAILURE' };
+		const initialState = {
+			error: undefined,
+			message: undefined,
+			tripCreated: '',
+		};
+		expect(tripRequestReducer({}, action)).toEqual(initialState);
+	});
+
+	it('should return initial sate', () => {
+		const action = { type: 'GET_LOCATIONS' };
+		const initialState = {};
+		expect(tripRequestReducer({}, action)).toEqual(initialState);
+	});
+	it('should return initial sate', () => {
```

Comment:
This and the above test cases, though are different, they carry the same test case description. It should be different.